cli/aws_orbit/data/toolkit/template.yaml

```diff
@@ -103,7 +103,7 @@ Resources:
             Action:
               - cloudformation:DescribeStacks
             Resource:
-              - arn:aws:cloudformation:${region}:${account_id}:stack/softwarelabs-remote-toolkit*
+              - arn:aws:cloudformation:${region}:${account_id}:stack/aws-codeseeder-toolkit*
```

Comment:
is this still called -toolkit? I thought I renamed it to -seedkit. I'll double check